content/applications/marketing/social_marketing/essentials/social_essentials.rst

```diff
@@ -0,0 +1,355 @@
+the :menuselection:`Social Marketing` app, click :menuselection:`Posts --> Create`.
+
+Either route reveals a blank post template page that can be customized and configured in a number
+of different ways.
+
+.. image:: social_essentials/social-create-post.png
+   :align: center
+   :alt: How to create a social media post directly through Odoo
+
+Post template
+-------------
+
+The post template page has many different options avaiable.
+
+'Your Post' section
+~~~~~~~~~~~~~~~~~~~
+
+The first option is the :guilabel:`Post on` field. This is where it's determined on what social
+media account(s), or on which website(s) via push notification, this post will be published.
+
+.. important::
+   In order for the :guilabel:`Push Notification` option to appear, make sure the :guilabel:`Enable
+   Web Push Notifications` feature is enabled in the *Website* app. To do that, navigate to
+   :menuselection:`Website --> Configuration --> Settings`, activate :guilabel:`Enable Web Push
+   Notifications`, fill out the corresponding fields, and click :guilabel:`Save`.
+
+Odoo automatically provides every available social media account that's been linked to the database
+as an option in this section, as well.
+
+.. note::
+   If a social media account hasn't been added as a stream to the *Social Marketing* application, it
+   will not appear as an option on the post template.
+
+Next, there's the :guilabel:`Message` field. This is where the main content of the post is created.
+
+Type the desired message for the post in this field. To the right, as the :guilabel:`Message` field
+is populated, Odoo displays visual samples of how the post will look on all the previously selected
+social media accounts from the :guilabel:`Post on` field above.
+
+.. tip::
+   Emojis can also be added directly to the text in the :guilabel:`Message` field. Just click the
+   "smiley face" icon, located on the line of the :guilabel:`Message` field to the far right.
+   Clicking that "smiley face" icon reveals a drop-down containing numerous emojis to choose from.
+
+If images are to be used in the post, click the :guilabel:`ATTACH IMAGES` link beneath the
+:guilabel:`Message` field, and Odoo reveals a pop-up window. In this pop-up, the desired image must
+be chosen, and then uploaded.
+
+A preview of the entire post, text and image (if applicable), is instantly displayed in the visual
+preview of the post.
+
+Next, there's the option to attach this post to a specific marketing campaign in the database in
+the :guilabel:`Campaign` field. Click the blank line next to :guilabel:`Campaign` to reveal the
+previously configured campaigns to choose from.
+
+.. tip::
+   A new campaign can be created on-the-fly, as well, by typing the name of the new campaign on the
+   blank :guilabel:`Campaign` field, and selecting :guilabel:`Create` from the drop-down. Or, select
+   :guilabel:`Create and edit` from the drop-down to further customize that newly-created campaign.
```

Comment:
Remove colloquialism.
Add slightly more specificity around the type of menu (i.e., field vs main vs. header menu).
```suggestion
.. tip::
   A new campaign can be created, as well, by typing the name of the new campaign on the blank
   :guilabel:`Campaign` field, and selecting :guilabel:`Create` from the drop-down field menu. Or,
   select :guilabel:`Create and edit` from the menu to further customize that newly-created
   campaign.
```